pipelines/vars/luna_upgrade_3.8.yml

```diff
@@ -0,0 +1 @@
+katello_upgrade_3.8.yml
```

Comment:
Puttin gthe name of a file in the file causes it to include the contents of the file?

Reply:
Nah, this is the GH ui being silly, it's a symlink as luna and katello upgrade paths are identical.